libs/payments/ui/src/lib/client/components/CheckoutForm.tsx

```diff
@@ -162,25 +166,31 @@ export function CheckoutForm({ readOnly, cart }: CheckoutFormProps) {
           )}
         </Form.Field>
       )}
-      <PaymentElement options={{ readOnly }} />
+      <PaymentElement
+        options={{
+          layout: {
+            type: 'accordion',
+            defaultCollapsed: false,
+            radios: false,
+            spacedAccordionItems: true,
+          },
+        }}
+      />
       {!isPaymentElementLoading && (
         <Form.Submit asChild>
-          <Localized id="next-new-user-submit">
-            <button
-              className="flex items-center justify-center bg-blue-500 font-semibold h-12 rounded-md text-white w-full p-4 mt-6 hover:bg-blue-700 aria-disabled:relative aria-disabled:after:absolute aria-disabled:after:content-[''] aria-disabled:after:top-0 aria-disabled:after:left-0 aria-disabled:after:w-full aria-disabled:after:h-full aria-disabled:after:bg-white aria-disabled:after:opacity-50 aria-disabled:after:z-30 aria-disabled:border-none"
-              type="submit"
-              aria-disabled={
-                !stripeFieldsComplete || !nonStripeFieldsComplete || loading
-              }
-            >
-              <Image
-                src={LockImage}
-                className="h-4 w-4 my-0 mx-3 relative top-0.5"
-                alt=""
-              />
-              Subscribe Now
-            </button>
-          </Localized>
+          <PrimaryButton
+            type="submit"
+            aria-disabled={
+              !stripeFieldsComplete || !nonStripeFieldsComplete || loading
+            }
+          >
+            <Image
+              src={LockImage}
+              className="h-4 w-4 my-0 mx-3 relative top-0.5"
+              alt=""
+            />
+            Subscribe Now
```

Comment:
Does this need to be l10n'ed? (I see we're translating the `placeholder=` above on L147).

Reply:
Ah good catch thank you. Deleted too much when I added PrimaryButton.